docs/release-notes/2024-12-19-new-feature-available-for-anti-fraud-providers-gift-card-transaction-analysis.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "New feature available for anti-fraud providers: Gift card transaction analysis"
+slug: "2024-12-19-new-feature-available-for-anti-fraud-providers-gift-card-transaction-analysis"
+hidden: false
+type: "added"
+createdAt: "2024-12-19T00:00:00.219Z"
+updatedAt: "2024-12-19T00:00:00.219Z"
+excerpt: "Gift card transactions can now be analyzed with anti-fraud."
+---
+
+Anti-fraud providers integrated with VTEX can now analyze payment transactions made with [gift cards](https://help.vtex.com/en/tutorial/gift-card--tutorials_995).
+
+## What has changed?
+
+The `allowAntifraudOnGiftCard` field was created within the [Anti-fraud Provider Protocol](https://developers.vtex.com/docs/guides/how-the-integration-protocol-between-vtex-and-antifraud-companies-works), allowing anti-fraud providers to indicate whether they want to process gift card transaction data in a VTEX store.
```

Comment:
Sobre o trecho: "allowing anti-fraud providers to indicate whether they **want** to process", me pareceupouco usual falar sobre o anti-fraud provider "querer (want)" processor a transação.

Sugeriria trocar para algo como: "whether they were designated / configured  to process gift card transaction" Ou atribuir a "vontade" (want) ao merchant e reestruturar a frase.